docs/deploying_an_application.md

```diff
@@ -0,0 +1,109 @@
+# Empire :: Deploying an Application
+
+Before using Empire successfully it's important to understand it's design and architecture. It's suggested you first read the `Features & Design Goals document`.
+
+There are a few key things which your applications, and overall architecture, will need to keep in mind in order to be successful with Empire.  Below is an exhaustive set of criteria your applications must meet in order to work with Empire.  It's not that long and by no means locks you in to Empire.  Structing your application in this way provides many benefits, only one of which is being able to be managed by Empire.
+
+For the impatient, there are two absolutes:
+
+- Your must create a `Procfile` which defines how to run your application
+- If you application answers requests over the network, it must listen on port 8080, or better, on the `$PORT` environment variable.
+
+To see an example application, you can look at [remind101/acme-inc]
+
+
+
+## Procfile
+
+Empire was modeled after the Heroku API.  As such, many concepts and commands are similar.  One concept which has bled over and which is a requirement is that of a [Procfile][procfiles].  For your application to actually work, you must include a Procfile at the root level of your appplication.
```

Comment:
So it turns out that a Procfile *isn't* necessarily required.  Empire tries to do the right thing when told to deploy an app without a Procfile.  Still, a Procfile is a lot better than hoping Empire gets it right.  This is probably fine here.